### DISCUSSION TITLE
Exit code 3

### DISCUSSION THREAD

**Discussion by gocfd** [nextfoam/baram#108] [Q&A]
Hello,

after several attempts I am unable to succesfully run the calculation for MRF case (k-epsilon turbulence model). Boundary conditions: mass flow inlet, pressure outlet.
I got his error message: C:\Program Files\BARAM\solvers\openfoam\bin\buoyantSimpleNFoam ended prematurely and may have crashed. exit code 3. Any ideas what could went wrong?

Thank you

**Comment by kdg5424** [comment]
Hi @gocfd. I'm Donggyu kim from NEXTFOAM.
Could you upload your case file? I'll check it.
Thank you

**Comment by gocfd** [comment]
Hi,

wetransfer link to files: https://we.tl/t-NIQgrfvVmF
Best regards.

**Comment by kdg5424** [comment]
Hi.

I checked your case file.

The log was meaning that calculation was diverged. So I suggest you two things.

First of all, the boundary is toooooo large. The diameter of the fan is around 120m.
Could you check it? After scaling the mesh 0.01times, the calculation works very well
You can scale the mesh on the tab **mesh-scaling**.
![image](https://github.com/user-attachments/assets/17846095-8e90-4f87-aae4-00ce3e2b2f97)



And second, the configuration of the interface is wrong. You should make it as **"interface"**, not **Thermo-Coupled Wall**.

![image](https://github.com/user-attachments/assets/2fd8beab-575b-46ff-a640-c229bb8c71dd)

![image](https://github.com/user-attachments/assets/f2c2cd8d-f96e-4e83-9f78-5043f090ac29)

Thanks

Best regards

Donggyu kim
